src/main/java/co/omise/requests/Request.java

```diff
@@ -70,6 +73,17 @@ public RequestBody getPayload() {
         return payload;
     }
 
+    /**
+     * Gets payload (request params).
+     *
+     * @return the payload as a readable string
+     */
+    public String getPayloadToString() throws IOException {
```

Comment:
@AnasNaouchi If we are adding, shouldn't we use somewhere?

Reply:
Its just a debugging function, we did not use it as we don't need to view the body in the actual code but every time a problem occurs with the payload we need to view it using this function.

Reply:
@AnasNaouchi wouldn't it be useful in a test to verify the JSON body is what we expect?

Reply:
It will not be what we want as we will have null values for keys hat are excluded

Reply:
But thinking back to your confusion earlier, if you saw a test that explicitly showed how the Java integration worked, wouldn't that have been helpful?

Reply:
Will add unit tests for that